BusTracker/Networking/RequestManager.swift

```
@@ -13,12 +13,15 @@ import Moya
 protocol RequestManagerDelegate {
     func receivedBusPositions(buses: [TransportPosition])
     func receivedTramPositions(trams: [TransportPosition])
+    func presentError(error: Error)
+    func hideError()
```

RequestManager shouldn't have any idea about presenting or hiding errors. All it knows is that it failed to load positions. Therefore you can replace this two methods with something similar to `requestPositionsFailed(error: Error)`.
`hideError` method is not needed - you can hide error when `receivedBusPositions` or `receivedTramPositions` is called.